DiverseSelector/methods/dissimilarity.py

```diff
@@ -36,16 +36,22 @@
 
 
 class MaxMin(SelectionBase):
-    """Selecting compounds using MaxMin algorithm.
-
-    Initial point is chosen as medoid center. The second point is
-    the furthest point away. All the following points are selected
-    using the rule:
-    1. Find minimum distance from every point to the selected ones.
-    2. Select a point the has the maximum distance among calculated
-       on the previous step.
-
-    The algorithm is described well here: https://doi.org/10.1002/qsar.200290002
+    """Selecting samples using MaxMin algorithm.
+
+    MaxMin is possibly the most widely used method for dissimilarity-based
+    compound selection. When presented with a dataset of samples, the
+    initial point is chosen as the dataset's medoid center. Next, the second
+    point is chosen to be that which is furthest from this initial point.
+    Subsequently, all following points are selected via the following
+    logic:
+
+    1. Find the minimum distance from every point to the already-selected ones.
+    2. Select the point which has the maximum distance among those calculated
+       in the previous step.
+
+    Notes
+    -----
+    The algorithm is described well here: Quant. Struct.-Act. Relat., 21 (2002). https://doi.org/10.1002/qsar.200290002
```

Comment:
I think putting it into the "References" filed is a better option. What do you think?

```rst
References
-----------
[1] Ashton, Mark, et al., Identification of diverse database subsets using property‐based and fragment‐based molecular descriptions, Quantitative Structure‐Activity Relationships 21.6 (2002): 598-604.
```

Reply:
I agree with this, and I'll change things accordingly. The only reason I had "Notes" previously was because this is how things are laid out in the [DirectedSphereExclusion](https://github.com/theochem/DiverseSelector/blob/main/DiverseSelector/methods/partition.py#L47) class, which Farnaz asked me to mimic. If we stick with "References", then that class should be updated to reflect this, too.